constants/jenkins.py

```diff
@@ -0,0 +1,14 @@
+QA_JENKINS = "qa_jenkins"
+QE_JENKINS = "qe_jenkins"
+JENKINS_URLS = {
```

Comment:
What is the purpose of having this dict ?
I see that having just the URLs as a list with a valid alias will help more.

Reply:
Got it.. I'm okay with this then